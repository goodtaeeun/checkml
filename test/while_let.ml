let while_example () =
  let x = 0 in
  while x < 1 do
    Printf.printf "This will print once since x is always 0.\n";
    exit 0 (* Ensures the while loop stops *)
  done
