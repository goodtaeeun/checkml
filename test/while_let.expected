File while_let.ml, line 3, column 2
[31mError[0m: Don't use while!

